calico-cloud/get-started/connect/install-automated.mdx

```diff
@@ -21,14 +21,14 @@ Using these credentials with standard kubectl and Helm installations lets you fu
 
 You can create client credentials and generate a Kubernetes secret to use for automated installations.
 
-1. Select the user icon <IconUser width="20"/> > **Client Credentials**, and then click **Add Client Credentials**.
-1. Click the `Add Client Credentials` button
+1. Select the user icon <IconUser width="20"/>** > Settings**.
+1. Under the **Client Credentials** tab, click **Add Client Credentials**
 1. In the **Add Client Credential** dialog, enter a name and click **Create**.
-   Your new client credentials will appear in the list on the **Manage Client Credentials** page.
+Your new client credentials will appear in the list on the **Manage Client Credentials** page.
```

Comment:
should it read `Your new client credential` instead of `Your new client credentials`?

Reply:
You're right to point out that my copy doesn't match the UI. But it's the UI that needs to change. Is that easily enough done?

"Credential" in this sense needs to be used as a plural noun. Imagine a sign-in page that says "Enter your credential and click **Save**.

Consider:
1) John has an apple. Mary has an apple. John and Mary have apples.
2) John has ~~a credential~~ credentials. Mary has ~~a credential~~ credentials. John and Mary have credentials.

Reply:
It seems this bit of technical jargon has its advocates: https://auth0.com/docs/api/management/v2/clients/post-credentials.

In any case, it's grey area, so I'll chalk this up to being a technical term, however jarring it is to my ear. Changes forthcoming.